client/js/initialize.js

```diff
@@ -1,2 +1,16 @@
-renderHeader();
-renderSearch();
+renderAppWithoutSession();
+renderAppWithSession();
+
+function renderAppWithoutSession() {
+    renderHeader();
+    renderSearch();
+    renderPlant(1);
```

Comment:
We probably need to make a decision here about how much info is open to users that aren't logged in. Do we want to offer full plant details? Or do we want to send them to a login page to be able to see the plant details page?